src/latex2sympy/units/unit_aliases.py

```diff
@@ -0,0 +1,353 @@
+from sympy import latex
+import sympy.physics.units.definitions.unit_definitions as sympy_units
+from sympy.physics.units.quantities import Quantity, PhysicalConstant
+from sympy.physics.units.systems.mks import all_units as mks_units, units as mks_base_units
+from sympy.physics.units.systems.mksa import all_units as mksa_units, units as mksa_base_units
+from sympy.physics.units.systems.si import all_units as si_units, units as si_base_units
+from latex2sympy.units.sie import all_units as sie_units
+from latex2sympy.units.prefixes import NEW_SI_PREFIXES, SI_PREFIXES, INFORMATION_SI_PREFIXES, BIN_PREFIXES, ALL_PREFIXES, PREFIX_ALIASES, prefix_unit
```

Comment:
@builtbywill Just a bit of import cleanup, PREFIX_ALIASES and json below aren't used.

Reply:
@tragessere yeah those are for when the debug json output section is uncommented